src/sentry/search/events/datasets/metrics.py

```diff
@@ -1848,9 +1849,38 @@ def _resolve_total_performance_score_function(
             for vital in vitals
         }
 
+        weights = {
+            vital: Function(
+                "if",
+                [
+                    Function(
+                        "isZeroOrNull",
+                        [
+                            Function(
+                                "countIf",
+                                [
+                                    Column("value"),
+                                    Function(
+                                        "equals",
+                                        [
+                                            Column("metric_id"),
+                                            self.resolve_metric(f"measurements.score.{vital}"),
+                                        ],
+                                    ),
+                                ],
+                            ),
+                        ],
+                    ),
+                    0,
+                    constants.WEB_VITALS_PERFORMANCE_SCORE_WEIGHTS[vital],
+                ],
+            )
+            for vital in vitals
+        }
+
         # TODO: Is there a way to sum more than 2 values at once?
         return Function(
-            "plus",
+            "divide",
```

Comment:
We divide by the maximum possible weight (which will be some value between 0 - 1.0 depending on which vitals are present). This will scale up the result if there are any vitals missing.